mc/include/alps/mc/mpiadapter.hpp

```diff
@@ -72,15 +72,15 @@ namespace alps {
                             alps::mpi::all_reduce(communicator,
                                                   has_count,
                                                   std::plus<size_t>());
-                    if (sum_counts == communicator.size()) {
+                    if ((int)sum_counts == communicator.size()) {
```

Comment:
Not a big deal, but why C-style cast `(int)` rather than `static_cast<int>()`, for consistency with other places?

Reply:
Lazyness, mainly. Will fix it.